package/src/components/Attachment/__tests__/Giphy.test.js

```diff
@@ -291,4 +267,39 @@ describe('Giphy', () => {
       expect(queryByTestId('giphy-attachment')).toBeTruthy();
     });
   });
+
+  it('should render a loading indicator in giphy image and when successful render the image', () => {
+    const { getByA11yLabel, getByAccessibilityHint } = render(
+      <OverlayProvider>
+        <Chat client={chatClient}>
+          <Channel channel={channel}>
+            <MessageList />
+          </Channel>
+        </Chat>
+      </OverlayProvider>,
+    );
+
+    fireEvent(getByA11yLabel('giphy-attachment-image'), 'onLoadStart');
+
+    expect(getByAccessibilityHint('image-loading')).toBeTruthy();
+
+    fireEvent(getByA11yLabel('giphy-attachment-image'), 'onLoadFiniah');
+
```

Comment:
You would like to expect here that the `getByAccessibilityHint('image-loading')` disappears